### PR TITLE
Feat/jwt auth

### LINKED ISSUE #3: Backstage - JWT Base Auth API
# Features

- [x] `api/v1` endpoint 
- [x] `GET /api/v1/health` API
- [x] JWT Auth
- [x] `POST /api/v1/login` API
- [ ] `PATCH/api/v1/token` API
- [x] `GET /api/v1/self` API

### PR DESCRIPTION
**Pull request by shadow3x3x3**
Fix #3

### CODE REVIEW

File: app/router/auth/auth_router.go [hunk 1]
```diff
@@ -1,4 +1,56 @@
...
+type LoginResp struct {
+	AccessToken  string `json:"access_token"`
+	RefreshToken string `json:"refresh_token"`
+}
+
+func (r Router) Login(c *gin.Context) {
+	api := api.New(c)
+
+	var payload LoginPayload
+	if err := c.ShouldBindJSON(&payload); err != nil {
```
Comment (Vince-Ku) [inline app/router/auth/auth_router.go:39]: Just out of curiosity, how often do you choose between those two?
`var payload LoginPayload`
`payload: = LoginPayload {}`

Comment (shadow3x3x3) [reply app/router/auth/auth_router.go:39]: Nothing special, just one of the styles. 🤣 

btw, this repo should follow as closely as possible to [Uber golang Guide](https://github.com/uber-go/guide/blob/master/style.md#use-var-for-zero-value-structs).

Comment (Vince-Ku) [reply app/router/auth/auth_router.go:39]: got it !

**Review by Vince-Ku — APPROVED** [review]
LGTM

**Pull request merged**